omnibus/config/software/librdkafka.rb

```diff
@@ -0,0 +1,42 @@
+name "librdkafka"
```

Comment:
Dependencies that don't go in the Agent packages should be added in https://github.com/DataDog/datadog-agent-buildimages/ (and for macOS to https://github.com/DataDog/datadog-agent-macos-build/blob/master/scripts/builder_setup.sh).

Just a note here that I'll leave for your consideration: It seems that you define a `default_version`, its `sha256` etc, but then proceed to install via vcpkg/yum/... This basically means that you get a random version that doesn't necessarily correspond to `default_version`. This might or might not be ok depending on librdkafka compatibility guarantees and I'll leave that up to you - I want to highlight this so that we know it's an intentional and explicit choice.

Reply:
I'm gonna have a deper look at their doc and try some stuff locally and I'll let you know. Thanks again for your help!